---
title :"问题列表二"
---

# 问题列表二

### 电竞大厅

* 产品分类： query接口查得到数据，view查不到：【已修复】

  * ![mark](http://7xlnr9.com1.z0.glb.clouddn.com/blog/20170805/174639214.png)

  * ![mark](http://7xlnr9.com1.z0.glb.clouddn.com/blog/20170805/174820725.png)

  * 新增报错：【已修复】

    ```
    IntegrityError: (1048, \\\"Column 'c_product_id' cannot be null\\\")\\n\",\"code\":-1,\"id\":null}"
    ```

     实际上是传了改参数的：

    ![mark](http://7xlnr9.com1.z0.glb.clouddn.com/blog/20170805/175240141.png)

* 资源管理，query接口无数据

  * 新增资源报错：【已修复】

    * ```
      "{\"message\":\"Traceback (most recent call last):\\n  File \\\"\\\/home\\\/wangjushidaigame\\\/rh\\\/video_dx_sichuan\\\/base_api\\\/views.py\\\", line 330, in save\\n    ret = instance.save()\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 589, in save\\n    force_update=force_update, update_fields=update_fields)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 617, in save_base\\n    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 698, in _save_table\\n    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 731, in _do_insert\\n    using=using, raw=raw)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/manager.py\\\", line 92, in manager_method\\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/query.py\\\", line 921, in _insert\\n    return query.get_compiler(using=using).execute_sql(return_id)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/sql\\\/compiler.py\\\", line 921, in execute_sql\\n    cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/utils.py\\\", line 65, in execute\\n    return self.cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/utils.py\\\", line 94, in __exit__\\n    six.reraise(dj_exc_type, dj_exc_value, traceback)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/utils.py\\\", line 65, in execute\\n    return self.cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/mysql\\\/base.py\\\", line 129, in execute\\n    return self.cursor.execute(query, args)\\n  File \\\"build\\\/bdist.linux-x86_64\\\/egg\\\/MySQLdb\\\/cursors.py\\\", line 174, in execute\\n    self.errorhandler(self, exc, value)\\n  File \\\"build\\\/bdist.linux-x86_64\\\/egg\\\/MySQLdb\\\/connections.py\\\", line 36, in defaulterrorhandler\\n    raise errorclass, errorvalue\\nIntegrityError: (1452, 'Cannot add or update a child row: a foreign key constraint fails (`video_dx_sichuan`.`t_resource`, CONSTRAINT `t_resource_ibfk_1` FOREIGN KEY (`cp_id`) REFERENCES `t_cps` (`id`))')\\n\",\"code\":-1,\"id\":null}"
      ```

    * 请求参数：

      ```
      zsjid=11&assid=11&c_name=11&cp_id=2&status=new&c_corps=11&c_commentary=11&c_match=11&c_time=2017-08-05+18%3A50%3A36&status=0&content=1&play_auth=1&img_url=&start_param=&play_count=1&praise_count=1&favor_count=1&c_sort=1
      ```

      ​

* 产品绑定资源：query接口无数据【已修复】

  * 资源无数据，无法绑定资源和产品【已修复】

  * 编辑时报错：

    ```
    "{\"message\":\"Traceback (most recent call last):\\n  File \\\"\\\/home\\\/wangjushidaigame\\\/rh\\\/video_dx_sichuan\\\/base_api\\\/views.py\\\", line 330, in save\\n    ret = instance.save()\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 589, in save\\n    force_update=force_update, update_fields=update_fields)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 617, in save_base\\n    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 679, in _save_table\\n    forced_update)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 723, in _do_update\\n    return filtered._update(values) > 0\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/query.py\\\", line 600, in _update\\n    return query.get_compiler(self.db).execute_sql(CURSOR)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/sql\\\/compiler.py\\\", line 1005, in execute_sql\\n    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/sql\\\/compiler.py\\\", line 787, in execute_sql\\n    cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/utils.py\\\", line 65, in execute\\n    return self.cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/utils.py\\\", line 94, in __exit__\\n    six.reraise(dj_exc_type, dj_exc_value, traceback)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/utils.py\\\", line 65, in execute\\n    return self.cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/mysql\\\/base.py\\\", line 129, in execute\\n    return self.cursor.execute(query, args)\\n  File \\\"build\\\/bdist.linux-x86_64\\\/egg\\\/MySQLdb\\\/cursors.py\\\", line 174, in execute\\n    self.errorhandler(self, exc, value)\\n  File \\\"build\\\/bdist.linux-x86_64\\\/egg\\\/MySQLdb\\\/connections.py\\\", line 36, in defaulterrorhandler\\n    raise errorclass, errorvalue\\nIntegrityError: (1452, 'Cannot add or update a child row: a foreign key constraint fails (`video_dx_sichuan`.`t_product_resource`, CONSTRAINT `t_product_resource_ibfk_2` FOREIGN KEY (`c_resource_id`) REFERENCES `t_resource` (`id`))')\\n\",\"code\":-1,\"id\":1273}"
    ```

    请求参数：

    ```
    id=1273&c_product=68&c_resource=73&c_sort=11&status=1
    ```

    model:

    ![mark](http://7xlnr9.com1.z0.glb.clouddn.com/blog/20170808/193800050.png)

    ​

* 计费ID配置：query接口无数据

  * 新增接口报错：

    ```
    "{\"message\":\"Traceback (most recent call last):\\n  File \\\"\\\/home\\\/wangjushidaigame\\\/rh\\\/video_dx_sichuan\\\/base_api\\\/views.py\\\", line 330, in save\\n    ret = instance.save()\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 589, in save\\n    force_update=force_update, update_fields=update_fields)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 617, in save_base\\n    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 698, in _save_table\\n    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 731, in _do_insert\\n    using=using, raw=raw)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/manager.py\\\", line 92, in manager_method\\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/query.py\\\", line 921, in _insert\\n    return query.get_compiler(using=using).execute_sql(return_id)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/sql\\\/compiler.py\\\", line 921, in execute_sql\\n    cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/utils.py\\\", line 65, in execute\\n    return self.cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/mysql\\\/base.py\\\", line 134, in execute\\n    six.reraise(utils.IntegrityError, utils.IntegrityError(*tuple(e.args)), sys.exc_info()[2])\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/mysql\\\/base.py\\\", line 129, in execute\\n    return self.cursor.execute(query, args)\\n  File \\\"build\\\/bdist.linux-x86_64\\\/egg\\\/MySQLdb\\\/cursors.py\\\", line 174, in execute\\n    self.errorhandler(self, exc, value)\\n  File \\\"build\\\/bdist.linux-x86_64\\\/egg\\\/MySQLdb\\\/connections.py\\\", line 36, in defaulterrorhandler\\n    raise errorclass, errorvalue\\nIntegrityError: (1048, \\\"Column 'cp_id' cannot be null\\\")\\n\",\"code\":-1,\"id\":null}"
    ```

    请求参数：

    ```
    cp_id=2&product_key=aa&product_name=1&money=1&cmboid=1&channelcode=1&related_products=76&memo1=1&memo2=1&memo3=1&status=1
    ```

    model:

    ![mark](http://7xlnr9.com1.z0.glb.clouddn.com/blog/20170808/194359923.png)

* 产品绑定计费ID：

  * 编辑接口报错：

    ```
    "{\"message\":\"Traceback (most recent call last):\\n  File \\\"\\\/home\\\/wangjushidaigame\\\/rh\\\/video_dx_sichuan\\\/base_api\\\/views.py\\\", line 330, in save\\n    ret = instance.save()\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 589, in save\\n    force_update=force_update, update_fields=update_fields)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 617, in save_base\\n    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 698, in _save_table\\n    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/base.py\\\", line 731, in _do_insert\\n    using=using, raw=raw)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/manager.py\\\", line 92, in manager_method\\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/query.py\\\", line 921, in _insert\\n    return query.get_compiler(using=using).execute_sql(return_id)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/models\\\/sql\\\/compiler.py\\\", line 921, in execute_sql\\n    cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/utils.py\\\", line 65, in execute\\n    return self.cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/utils.py\\\", line 94, in __exit__\\n    six.reraise(dj_exc_type, dj_exc_value, traceback)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/utils.py\\\", line 65, in execute\\n    return self.cursor.execute(sql, params)\\n  File \\\"\\\/usr\\\/local\\\/python\\\/lib\\\/python2.7\\\/site-packages\\\/Django-1.7.9-py2.7.egg\\\/django\\\/db\\\/backends\\\/mysql\\\/base.py\\\", line 129, in execute\\n    return self.cursor.execute(query, args)\\n  File \\\"build\\\/bdist.linux-x86_64\\\/egg\\\/MySQLdb\\\/cursors.py\\\", line 174, in execute\\n    self.errorhandler(self, exc, value)\\n  File \\\"build\\\/bdist.linux-x86_64\\\/egg\\\/MySQLdb\\\/connections.py\\\", line 36, in defaulterrorhandler\\n    raise errorclass, errorvalue\\nIntegrityError: (1452, 'Cannot add or update a child row: a foreign key constraint fails (`video_dx_sichuan`.`t_product_bind_id`, CONSTRAINT `t_product_bind_id_ibfk_2` FOREIGN KEY (`c_productid_id`) REFERENCES `t_product_id` (`id`))')\\n\",\"code\":-1,\"id\":null}"
    ```

    请求参数：

    ```
    c_product=59&c_productid=73&status=0
    ```

    model:

    ![mark](http://7xlnr9.com1.z0.glb.clouddn.com/blog/20170808/201502892.png)

    ​

* 订购配置： query接口无数据【梁说暂时不做此功能】

  * 之前看四川现网此功能报500错误，无法使用
  * 重庆现网后台此功能报500错误，亦无法使用

* 拦截页面图片配置：query接口无数据【梁的文档中无此功能】

  * 重庆现网后台无该项功能
  * 之前四川现网看时由于没有数据，无法确定编辑与新增的字段

* 电竞导入数据： 接口404

  * ![mark](http://7xlnr9.com1.z0.glb.clouddn.com/blog/20170808/201646996.png)
  * ![mark](http://7xlnr9.com1.z0.glb.clouddn.com/blog/20170808/201718622.png)

  ​